Aquality.Selenium.Core/src/Aquality.Selenium.Core/Localization/LocalizationManager.cs

```diff
@@ -11,13 +11,13 @@ namespace Aquality.Selenium.Core.Localization
     public class LocalizationManager
     {
         private const string LangResource = "Resources.Localization.{0}.json";
-        private readonly JsonFile localizationFile;
+        private readonly JsonSettingsFile localizationFile;
```

Comment:
It can be ISettingFile

Reply:
decided to do it in #23

Reply:
I mean you can define type of readonly field as ISettingFile

Reply:
Yes, but I am not sure it would be correctly in this case , because we use definitely JsonSettingsFile for LocalizationManager